server/controllers/inventory/depots/index.js

```diff
@@ -234,3 +234,30 @@ function detail(req, res, next) {
     .catch(next)
     .done();
 }
+
+// Get depots by their uuids
+function getDepotGroup(req, res, next) {
+  const { depotUuids } = req.query;
+  const uuidGroup = [].concat(depotUuids);
+
+
+  let uuids = '';
+  uuidGroup.forEach((depotUuid, index) => {
+    if (index === 0) {
+      uuids = `HUID('${depotUuid}')`;
```

Comment:
Remember that passing in raw values always exposes you to SQL injection attacks.  You should always escape values before putting them into a string to give to the database.